docker/Dockerfile

```diff
@@ -17,8 +17,12 @@ RUN    apt-get update \
 
 # Pass in the VARNISH_VERSION build arg to specify the version of Varnish to install
 ARG VARNISH_VERSION=76
-RUN curl -sSf "https://packagecloud.io/install/repositories/varnishcache/varnish${VARNISH_VERSION}/script.deb.sh" | sudo bash \
-    && (sudo apt-get install -y varnish-dev || sudo apt-get install -y libvarnishapi-dev)
+RUN set -ex; \
+    curl -sSf "https://packagecloud.io/install/repositories/varnishcache/varnish${VARNISH_VERSION}/script.deb.sh" | sudo bash; \
+    echo 'Package: varnish varnish-dev\nPin: origin "packagecloud.io"\nPin-Priority: 1001' > /etc/apt/preferences.d/varnish; \
```

Comment:
I guess we will need some sort of an `if` statement here to dynamically decide how to install each version - unless you think this will work for all versions?

Reply:
this will work for all packages coming from packagecloud, which is what we want to use